src/w3emc/v2.2.0/src/makefile_d

```diff
@@ -4,7 +4,7 @@ include ../../../../macros.make
 
 version   = v2.2.0
 tarname   = w3emc
-precision = d
+precision = 4
```

Comment:
Why is the precision changed from "d" to "4" in makefile_d?

Reply:
Good catch! That was definitely an error. I had some fumbling when I was trying to negotiate the different naming conventions for all the different libraries; originally I started out trying to standardize all the library names but then I thought that would be too drastic of a change for this pull request.

As I mentioned in the commit message, I noticed something weird about the previous libw3emc build. Even though the library was built with the name "libw3emc_d.a", this was actually built with single/4-byte precision reals, at least for the Cheyenne macros files. I have changed it so that the expected precision corresponds to the appropriate filename, as now both precisions are built. This may have some impact on UFS results however, since it (perhaps unexpectedly) was previously using that library compiled for single precision.

If you think this was an inappropriate change, please let me know.